### ISSUE TITLE
Email subject doesn't match body

### ISSUE THREAD

**Issue by brynwhyman** [silverstripe/silverstripe-mfa#259]
### Overview

Emails sent to record MFA actions appear to have descrepencies between the email subject and email content.

### Task

1. Test all emails to ensure the subject and body are correct for each action.
2. Resolve any bugs identified - there's at least one (below).
3. There are unit tests to cover the subject as well as the email body

### Example bug:

Subject: 
`A multi-factor authentication method was added to your account`

Email body: 
```
Hi x

You have removed the Authenticator app authentication method from your account at x.

If you did not take this action, please contact your system administrator immediately.
```

**Comment by NightJar** [comment]
PR https://github.com/silverstripe/silverstripe-mfa/pull/276

**Comment by NightJar** [comment]
Fixed in #276

**Comment by ScopeyNZ** [comment]
Closed by #276
